l10n_it_fiscalcode/README.rst

```diff
@@ -0,0 +1,117 @@
+    :target: https://translation.odoo-community.org/projects/l10n-italy-14-0/l10n-italy-14-0-l10n_it_fiscalcode
+    :alt: Translate me on Weblate
+.. |badge5| image:: https://img.shields.io/badge/runbot-Try%20me-875A7B.png
+    :target: https://runbot.odoo-community.org/runbot/122/14.0
+    :alt: Try me on Runbot
+
+|badge1| |badge2| |badge3| |badge4| |badge5| 
+
+**Italiano**
+
+Questo modulo aggiunge al partner il campo codice fiscale e permette di calcolarne il suo valore.
+
+**English**
+
+This module extends the functionality of partner allowing you to compute its fiscal code.
+
+**Table of contents**
+
+.. contents::
+   :local:
+
+Installation
+============
+
+**Italiano**
+
+Installare il modulo Python: `codicefiscale <https://pypi.python.org/pypi/codicefiscale>`__.
+
+**English**
+
+Install the Python package: `codicefiscale <https://pypi.python.org/pypi/codicefiscale>`__.
+
+Usage
+=====
+
+**Italiano**
+
+Sulla scheda del partner fare clic sul pulsante *Calcola CF* per aprire la procedura guidata relativa al calcolo
+del codice fiscale.
+
+**English**
+
+Go to Partner and click "Compute FC" in order to open the wizard related to the fiscal code computation.
+
+Bug Tracker
+===========
+
+Bugs are tracked on `GitHub Issues <https://github.com/OCA/l10n-italy/issues>`_.
+In case of trouble, please check there if your issue has already been reported.
+If you spotted it first, help us smashing it by providing a detailed and welcomed
+`feedback <https://github.com/OCA/l10n-italy/issues/new?body=module:%20l10n_it_fiscalcode%0Aversion:%2014.0%0A%0A**Steps%20to%20reproduce**%0A-%20...%0A%0A**Current%20behavior**%0A%0A**Expected%20behavior**>`_.
+
+Do not contact contributors directly about support or help with technical issues.
+
+Credits
+=======
+
+Authors
+~~~~~~~
+
+* Link IT s.r.l.
+* Apulia Software
+* Odoo Italia Network
+
+Contributors
+~~~~~~~~~~~~
+
+* Davide Corio
+* Luca Subiaco <subluca@gmail.com>
+* Simone Orsi <simone.orsi@domsense.com>
+* Mario Riva <mario.riva@agilebg.com>
+* Mauro Soligo <mauro.soligo@katodo.com>
+* Giovanni Barzan <giovanni.barzan@gmail.com>
+* Lorenzo Battistini <lorenzo.battistini@agilebg.com>
+* Roberto Onnis <onnis.roberto@gmail.com>
+* Franco Tampieri
+* Andrea Cometa <info@andreacometa.it>
+* Andrea Gallina
+* Matteo Bilotta <mbilotta@linkgroup.it>
+* Giuseppe Borruso - Dinamiche Aziendali srl <gborruso@dinamicheaziendali.it>
+* Alex Comba <alex.comba@agilebg.com>
+
+Maintainers
+~~~~~~~~~~~
+
+This module is maintained by the OCA.
+
+.. image:: https://odoo-community.org/logo.png
+   :alt: Odoo Community Association
+   :target: https://odoo-community.org
+
+OCA, or the Odoo Community Association, is a nonprofit organization whose
+mission is to support the collaborative development of Odoo features and
+promote its widespread use.
+
+This module is part of the `OCA/l10n-italy <https://github.com/OCA/l10n-italy/tree/14.0/l10n_it_fiscalcode>`_ project on GitHub.
```

Comment:
Version 13.0, not 14.0

Reply:
Changed it now although but will regenerate it in merge process.